### ISSUE TITLE
S4ND layer and code

### ISSUE THREAD

**Issue by RoyHirsch** [HazyResearch/state-spaces#79]
Thank you for the wonderful and insightful repo, are there any plans on releasing the code for S4ND layer?

Roy

**Comment by albertfgu** [comment]
Yes, I'm working on a V4 release. I'm not sure when the full release will be done, but I'll start pushing pieces to a branch soon so the S4ND code should be available relatively soon.
